bind/files/debian/named.conf.options

```diff
@@ -1,7 +1,7 @@
 {%- set key_directory = salt['pillar.get']('bind:lookup:key_directory', key_directory) %}
 
 options {
-        directory "/var/cache/bind";
+        directory "{{ named_directory }}";
 
```

Comment:
Debian has `named_directory` set to `/var/cache/bind/zones` [in map.jinja](https://github.com/saltstack-formulas/bind-formula/blob/master/bind/map.jinja#L15). Not sure if **that** value is correct. Just noticing it's different than the hardcoded value. Won't it break things?

Reply:
It shouldn't break things, but you're right, the value should be `/var/cache/bind`.

Also the fedora and centos failures were related to Redhat os_family specific bug, which I've tracked down and have fixed; just running through the tests as I type (I didn't run these before, but have the requisite dependencies sorted now; sorry for not running them before!)